options.go

```diff
@@ -68,3 +70,10 @@ func WithFormatter(f Formatter) LoggerOption {
 		l.formatter = f
 	}
 }
+
+// WithLevelFromString sets level with given parameter.
+func WithLevelFromString(level string) LoggerOption {
+	return func(l *logger) {
+		l.level = ParseLevel(level)
+	}
+}
```

Comment:
This is no longer needed as you can use `WithLevel` instead. for example `WithLevel(ParseLevel("info"))`

Reply:
You are right. I removed it.

Reply:
It is ready to review and merge. Could you check the PR?